src/ui/IngredientForm.tsx

```diff
@@ -0,0 +1,24 @@
+import { FieldArray } from 'formik';
+import Ingredient from './Ingredient';
+import { FC } from 'react';
+
+const IngredientForm: FC = () => {
+  return (
+    <FieldArray
+      name="ingredients"
+      render={(arrayHelpers) => {
+        return arrayHelpers.form.values.ingredients.map((_, index) => {
+          return (
+            <Ingredient
+              index={index}
+              arrayHelpers={arrayHelpers}
+              key={`ingredient.${index * 1000}`}
```

Comment:
you should be able to just use ingredient here since ingredients should be unique in a recipe